specs/merge/fork-choice.md

```diff
@@ -71,16 +59,14 @@ def finalize_block(self: ExecutionEngine, block_hash: Hash32) -> bool:
 ### `TransitionStore`
 
 ```python
-@dataclass
-class TransitionStore(object):
+class TransitionStore(Container):
     terminal_total_difficulty: uint256
 ```
 
 ### `PowBlock`
 
 ```python
-@dataclass
```

Comment:
Suggest keeping `@dataclass` because this data is not serialized nor hashed. This conforms to our prior usage of `@dataclass` with the fork choice `Store`

Reply:
FYI it has changed to SSZ class so that we can provide serialized test vectors.